test/hook_SUITE.erl

```diff
@@ -68,6 +82,18 @@ run_testcases_fail_based_on_counter_repeat3(_Config) ->
     [{groups_summary, {0, 1}},
      {eventually_ok_tests, 0},
      {end_per_testcase_failures, 0}] = GrSum,
+    ["CT results:",
+     "    0 groups passed",
+     "    1 groups failed",
+     "    0 tests eventually passed",
+     "    0 tests with end_per_testcase failed",
+     "    6 tests passed",
+     "    3 tests failed","    0 tests skipped by user",
+     "    0 tests skipped automatically",
+     "No test groups were executed successfully",
+     "Failing the test due to failed rerun groups",
+     "Failing the test due to failed test cases",
+     "CODE=53"] = summarise_results(),
```

Comment:
Why is the code 53 even though CT results above show lower numbers?

Reply:
Oh, it is `erlang:halt(100405)`, but error code overflows.
```
1> erlang:halt(100405).
exits
echo $?
53
```